.github/workflows/build-and-test-pr.yml

```diff
@@ -3,6 +3,9 @@ on:
   merge_group:
     types: [checks_requested]
   pull_request:
+    branches-ignore:
+        - 'smartling-content-updated*'
+        - 'smartling-translation-completed*'
```

Comment:
should be `**` not `*`

From the [documentation](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet):

> `*`: Matches zero or more characters, but does not match the / character

The branch for https://github.com/LedgerHQ/ledger-live/pull/9816 is `smartling-translation-completed-aimeothvv6qo-from-feat/aptos-token-integration`